Sources/HaCWebsiteLib/ViewModels/LandingPage/LandingFeatures/GitWorkshop.swift

```diff
@@ -0,0 +1,19 @@
+import HaCTML
+import Foundation
+
+extension LandingFeatures {
+  static var gitWorkshop: LandingFeature {
+    return EventFeature(
+      startDate: Date.from(year: 2017, month: 11, day: 1, hour: 19, minute: 00),
+      endDate: Date.from(year: 2017, month: 11, day: 1, hour: 20, minute: 30),
+      eventLink: "https://www.facebook.com/events/1906244696304681",
```

Comment:
Do we possibly want two links? Like a pre-event link (for the Facebook event), and the on-event link (for the Workshop page?)

Apart from that, this all looks good, and is already a massive improvement over what we have!

Reply:
See `liveLink`